satellite/satellitedb/database.go

```diff
@@ -41,7 +43,9 @@ func New(log *zap.Logger, databaseURL string) (satellite.DB, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	if !strings.Contains(source, "ApplicationName") {
+		source = fmt.Sprintf("%s?%s", source, "ApplicationName=Satellite")
```

Comment:
the `source` may already contain query parameters and the driver can here be sqlite as well